src/api/models/index.ts

```diff
@@ -0,0 +1 @@
+export { default as SingleLocation } from './singleLocation-model';
```

Comment:
SingleLocation is already the default export in that file.
```suggestion
export SingleLocation from './singleLocation-model';
```

Reply:
No { default as ... } is correct. If they were exporting a named export as SingleLocation then yes you would export your way, but in the SingleLocation file it is exported as default so it needs to be named here in the index.